<?xml version="1.0"?>

<launch>
    
    <!-- path_plan data-->
    <node pkg="stauto_control" type="path_plan_node" name="path_plan_node">
    </node>

    <!-- global_path data-->
    <node pkg="stauto_sensor" type="global_path.py" name="global_path_node" >
    </node>

    <!-- control -->
    <node pkg="stauto_control" type="control.py" name="control_node" output="screen">
    </node>

    <!-- GPS data-->
    <!-- <node pkg="stauto_sensor" type="get_gps_data.py" name="gps_data" > 
        <param name="gps_port" type="str" value="$(arg gps_port)"/>
    </node> -->

    <!-- Velodyne data-->
    <!-- <include file="$(find pointcloud_to_laserscan)/launch/sample_node.launch"> 
    </include> -->

    <!-- <include file="$(find velodyne)/velodyne_pointcloud/launch/VLP16_points.launch"> 
    </include> -->

    <!-- RVIZ -->
    <!-- <node name="rviz" pkg="rviz" type="rviz"/> -->

</launch>
